web/src/js/components/Modes.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import HTTPS from "./Modes/HTTPS";
+import Local from "./Modes/Local";
+import Wireguard from "./Modes/Wireguard";
+import Reverse from "./Modes/Reverse";
+
+export default function Modes() {
+    return (
+        <div style={{ padding: "1em 2em" }}>
```

Comment:
Nit: I'd just go with a className for the mode tab and then use `.modes h3` instead of `.modes-subheader` below.